Modelica_ResultCompare/CsvFile.cs

```diff
@@ -313,7 +313,12 @@ public Report CompareFiles(Log log, CsvFile csvBase, string sReportPath, ref Opt
                     else
                         log.WriteLine(LogLevel.Debug, "The resolution of the base x-axis is good.");
 
-                    size = new TubeSize(reference, true);
+                    // The actual nominal attribute should be used, but is unfortunately unavailable in the CSV files.
+                    // A default nominal value of 0.001 was chosen as a compromise between having many false negatives
+                    // and passing wrong result files.

```

Comment:
The impulse for this change is https://github.com/modelica/ModelicaStandardLibrary/issues/4421. IMO there shall be a link to that issue in the code's comment to better understand the idea.

Reply:
Good point. See #72